### ISSUE TITLE
[package] qt/6.x.x: brotli versions conflict with freetype

### ISSUE THREAD

**Issue by evilenzo** [conan-io/conan-center-index#20451]
### Description

When I try to install qt/6.x.x I get such error
```
ERROR: Version conflict: freetype/2.13.0->brotli/1.1.0, qt/6.4.2->brotli/1.0.9.
```
Looks like it's a conflict inside qt dependencies. I think it's bad?

### Package and Environment Details

* Package Name/Version: **qt/6.4.2**
* Operating System+version: **Windows 10 22H2**
* Compiler+version: **MSVC**
* Conan version: **conan 2.0.13**
* Python version: **Python 3.10.11**

### Conan profile

```ini
Host profile:
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=23
compiler.runtime=dynamic
compiler.runtime_type=Release
compiler.version=193
os=Windows

Build profile:
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=23
compiler.runtime=dynamic
compiler.runtime_type=Release
compiler.version=193
os=Windows
```

### Steps to reproduce

```python
from conan import ConanFile, tools
from conan.tools.cmake import cmake_layout, CMake


class HeatEngine(ConanFile):
    settings = "os", "compiler", "build_type", "arch"

    generators = "CMakeDeps", "CMakeToolchain"

    def layout(self):
        cmake_layout(self)

    def requirements(self):
        self.requires("qt/6.4.2")
```

### Logs

No logs required

**Comment by ericLemanissier** [comment]
This will be fixed by https://github.com/conan-io/conan-center-index/pull/20186 which just miss an approval

**Comment by evilenzo** [comment]
> This will be fixed by #20186 which just misses an approval

Didn't see that one, sorry for duplicate
